### PR TITLE
Queue migration disable legacy orchestrations removed

### PR DESCRIPTION
**Pull request by MiroCillik**
Changes:

- no need to disable legacy orchestrations after queue migration - it's done in the migration tool component

### CODE REVIEW

File: src/Keboola/Console/Command/CheckProjectBeforeQueueMigration.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                $triggers[$orchestration['id']] = $foundTriggers;
+            }
+        }
+
+        return $triggers;
+    }
+
+    private function findOrchestratorServiceUrl(StorageClient $storageClient): string
+    {
+        $index = $storageClient->indexAction((new IndexOptions())->setExclude(['components']));
```
Comment (odinuv) [inline src/Keboola/Console/Command/CheckProjectBeforeQueueMigration.php:79]: getServiceUrl

File: src/Keboola/Console/Command/CheckProjectBeforeQueueMigration.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+use Exception;
+use Keboola\Orchestrator\Client as OrchestratorClient;
+use Keboola\StorageApi\Client as StorageClient;
+use Keboola\StorageApi\Options\IndexOptions;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CheckProjectBeforeQueueMigration extends Command
```
[the remark this answers is not in the document]
Comment (MiroCillik) [reply src/Keboola/Console/Command/CheckProjectBeforeQueueMigration.php:16]: Jo, na to som aj zabudol, to som si napisal, ked som migroval HCI a prislo mi vtedy jednoduchsie to strcit sem :)

[the remark this answers is not in the document]
Comment (MiroCillik) [reply src/Keboola/Console/Command/CheckProjectBeforeQueueMigration.php:16]: To nemalo byt v tomto PR :)

**Comment by MiroCillik** [comment]
Presunul som ten check na triggery do komponenty.

**Review by odinuv — APPROVED** [review]

**Pull request merged**